src/components/Navigator/NavigatorCard.vue

```diff
@@ -32,18 +32,23 @@
         @keydown.up.exact.capture.prevent="focusPrev"
         @keydown.down.exact.capture.prevent="focusNext"
       >
-        <Reference
-          v-if="technology"
-          :id="INDEX_ROOT_KEY"
-          :url="technologyPath"
-          :class="['technology-title', { 'router-link-exact-active': isTechnologyRoute }]"
-          @click.alt.native.prevent="toggleAllNodes"
-        >
-          <h2 class="card-link">
-            {{ technology }}
-          </h2>
-          <Badge v-if="isTechnologyBeta" variant="beta" />
-        </Reference>
+        <slot name="navigator-title" className="navigator-title">
```

Comment:
```suggestion
        <slot name="navigator-title">
```

I'm not sure you can add classes to the slot element directly, right? And it looks like you're already adding the class to the reference component below anyway. Also isn't `className` a react/jsx thing? 😁